src/hal/getpinmap_ttgo_lora32_v2.1.cpp

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
The module should have a simple descriptive header to match the other files in this group.